### DISCUSSION TITLE
Issues with GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by chace-carey** [community/community#110772] [GitHub Education]
Hello, I'm a third year student at the University of Rhode Island and I'm trying to sign up for the Student Developer Pack but I'm running into some issues. I've gone through the application process multiple times, even waiting weeks between attempts to see if one of them went through but it seems like my applications aren't even going through. My account uses my school email, I've uploaded a picture of my ID both as a JPEG and as a PNG, and I've included both my name and a small description in my profile which are all problems that other posts have brought up. Other posts also say that the top of the application page should include your verification status but no such status is displayed on my page. Whenever I press the process application button in takes a few seconds and then seems to just refresh the page and that is all. I'm hoping to fix this issue because I would really like to try out the Student Developer Package for myself since I've heard such good things from my peers.

Thank you in advance for a response.

**Comment by khanshaib11** [comment]
same happened with me, u have to re-upload the file with different angle for the system to accept it.
